options.go

```diff
@@ -365,3 +365,42 @@ type ListOptions struct {
 type CleanOptions struct {
 	Dir bool
 }
+
+// GrepOptions describes how a grep should be performed.
+type GrepOptions struct {
+	// Pattern is the match pattern.
+	Pattern string
```

Comment:
Itstead of accepting a string, I would prefer a `*regexp.Regexp` with this the people can speed that the process with precompilled expressions.